tiles/src/main/java/com/protomaps/basemap/layers/Roads.java

```diff
@@ -35,7 +35,7 @@ public void processFeature(SourceFeature sourceFeature, FeatureCollector feature
           shield_text = shield_text.replaceAll("I ", "");
           network_val = "US:I";
         } else {
-          network_val = "other";
+          network_val = network_val;
```

Comment:
This will allow other shields besides `US:I` and `US:US` instead of blocking them...